src/main/java/com/github/games647/flexiblelogin/FlexibleLogin.java

```diff
@@ -131,12 +134,18 @@ public void onInit(GameInitializationEvent initEvent) {
     }
 
     private void registerCommands() {
-        commandManager.register(this, injector.getInstance(LoginCommand.class).buildSpec(), "login", "log");
+        List<String> loginAliases = new ArrayList<>(Arrays.asList("login", "log"));
```

Comment:
`new ArrayList` ist redundant. `Arrrays.asList` already creates an ArrayList.

Reply:
Yes, but List returned by Arrays.asList() is immutable.

Reply:
Okay that's correct. Sorry.